avocado/core/nrunner.py

```diff
@@ -73,6 +81,23 @@ class PodmanSpawner(BaseSpawner):
     METHODS = [SpawnMethod.STANDALONE_EXECUTABLE]
     IMAGE = 'fedora:31'
 
+    @staticmethod
+    def is_alive(task):
```

Comment:
Suggestion: `is_task_alive()`. Also docstring is missing.

Reply:
Then, it may also make sense to rename `spawn()`? I mean, renamed it to `spawn_task()`? I have the feeling that the spawn classes will be limited to tasks, so it seems like a bit of overkill. What do you think?

Reply:
For me makes sense, because `is_alive` is a method of a Spawner. And when I do something like:

```
spawner.is_alive(foo)
```
It is not obvious that foo is a task. So `is_task_alive()` would be better. Even when limited to tasks.